pkg/user/appointee/remove.go

```diff
@@ -56,18 +60,88 @@ func removeAppointeePermission(
 	}
 	cliCtx.App.Metadata["network"] = config.ChainID.String()
 	permissionWriter, err := generator(logger, config)
+	removePermissionRequest := elcontracts.RemovePermissionRequest{
+		AccountAddress:   config.AccountAddress,
+		AppointeeAddress: config.AppointeeAddress,
+		Target:           config.Target,
+		Selector:         config.Selector,
+		WaitForReceipt:   true,
+	}
+	if config.Broadcast {
+
+		if err != nil {
+			return err
+		}
+		err = broadcastRemoveAppointeeCallData(ctx, permissionWriter, config, removePermissionRequest)
+		if err != nil {
+			return err
+		}
+	} else {
+		err = printRemoveAppointeeCallData(logger, permissionWriter, config, removePermissionRequest)
+	}
+	return err
+}
+
+func printRemoveAppointeeCallData(
```

Comment:
`printRemoveAppointeeCallData` - it's not just printing calldata so change name as the other comment sugggestions.